pkg/katautils/create.go

```diff
@@ -35,6 +35,22 @@ var systemdKernelParam = []vc.Param{
 		Key:   "systemd.mask",
 		Value: "systemd-networkd.socket",
 	},
+	{
+		Key:   "systemd.mask",
+		Value: "systemd-journald.service",
```

Comment:
I think this will break the agent tracing PR - https://github.com/kata-containers/agent/pull/415, specifically the `kata-journald-host-redirect.service` servivce. That isn't technically part of tracing, but allows guest journal messages to propagate out of the VM when vsock is enabled.

Reply:
I guess we can do something like this

```go
if configuration.enable_tracing == false {
    // Disable systemd udev and journald
}
```